doc/sphinx/source/community/detailed_release_procedure.rst

```diff
@@ -0,0 +1,216 @@
+Note that the ``summarize.py`` script needs to be rerun if recipe runs were added or deleted
+from your testing directory.
+
+Link the overview webpage to the release issue.
+This makes it much easier to ask for feedback from recipe developers and analyse failures.
+
+Results produced with the final ESMValCore release candidate should be put in a VM directory
+named after the version number, e.g. ``v2.x.x``. 
+Once the release process is over, test results produced with previous release candidates can be deleted to save space on the VM.
+
+.. note::
+
+  If you wrote recipe runs output to Levante's `/scratch` partition, be aware that
+  the data will be removed after two weeks, so you will have to quickly move the 
+  output data to the VM, using the `nohup` command above.
+
+Running the comparison
+----------------------
+
+To compare the newly produced output from running all recipes, follow these steps below.
+
+Access to the DKRZ esmvaltool VM, then install miniconda on the VM, and
+if you have a Miniconda installer already downloaded in your Levante $HOME
+
+.. code-block:: bash
+
+  scp Miniconda3-py39_4.12.0-Linux-x86_64.sh user@esmvaltool.dkrz.de:/mnt/esmvaltool_disk2/work/<username>
+
+.. warning::
+
+  conda environments should not be created in the home directory because it is on a very small disk,
+  but rather in a directory with your username under `/mnt/esmvaltool_disk2/work/<username>`
+
+Next, we need to set up the input files
+
+The `/work` partition is visible by the VM so you can run the compare tool straight on the VM.
+
+The steps to running the compare tool on the VM are the following:
+
+- run date: log the run date here
+- conda env: log the name of the conda environment you are using
+- ESMValTool branch: log the name of the code branch you are using (e.g. `v2.8.x`)
+- prerequisite - install `imagehash`: `pip install imagehash`
+- reference run (v2.7.0; previous stable release): `export reference_dir=/work/bd0854/b382109/v270` (contains `preproc/` dirs too, 122 recipes)
+- current run (v2.8.0): `export current_dir=path_to_current_run`
+- command to run: 
+
+. code-block:: python
+
+  nohup python ESMValTool/esmvaltool/utils/testing/regression/compare.py --reference $reference_dir --current $current_dir > compare_v280_output.txt
+
+Copy the comparison txt file to the release issue.
+Some of the recipes will appear as having identical output to the one from previous release. However, others
+will need human inspection. Ask the recipe maintainers (`@ESMValGroup/esmvaltool-recipe-maintainers`_) and ESMValTool Development Team (`@ESMValGroup/esmvaltool-developmentteam`_) to provide assistance in checking the results.
+Here are some guidelines on how to perform the human inspection:
+
+- look at plots from current run vs previous release run: most of them will be identical, but if Matplotlib
+  has changed some plotting feature, images will have slightly different metadata so the comparison script will report them
```

Comment:
It would be good to note here that the comparison is actually smart enough to overlook small differences, or even better link the documentation of the comparison script

Reply:
Agreed. Done in 62e7f7e5e